### PR TITLE
Split nginx conf

### PR DESCRIPTION
**Pull request by Krakinou**
## Problem

- *Nginx file will be changed by any upgrade, this does not allows user to make change easily to the api path*

## Solution

- *Split the file in two and update only the basis one by default*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Krakinou** [comment]
!testme

**Comment by Tagadda** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -87,6 +87,7 @@ ynh_systemd_action --service_name=$app --action="stop" --log_path="/var/log/$app
 ynh_script_progression --message="Updating NGINX web server configuration..."
 
 nginx_conf_path=/etc/nginx/conf.d/$old_domain.d/$app.conf
+nginx_api_conf_path=/etc/nginx/conf.d/$domain.d/api_$app.conf
```
Comment (Tagadda) [inline scripts/change_url]:
```suggestion
nginx_api_conf_path=/etc/nginx/conf.d/$old_domain.d/api_$app.conf
```

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
Does not fail on my local packagee check??
!testme

**Comment by Krakinou** [comment]
Go figure...

**Pull request merged**